WebView/app/src/main/assets/index.html

```diff
@@ -23,7 +23,7 @@
     <label for="location">Choose a location:</label>
     <select name="location" id="location" onchange="getData()">
       <option value="newYork">New York</option>
-      <option value="sanFrancisco">San Francisco</option>
+      <option id= "SF" value="sanFrancisco">San Francisco</option>
```

Comment:
If you want to use "SF" here, I think it would be good to use "NY(C)" for New York and "LDN" for London.